examples/simple_static_task/static_test_prolific_script.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
```

Comment:
Now that this is all functional, I'm not sure what the _correct_ thing for us to be doing to demonstrate different providers will be, but I don't think it's going to be to have a different script in here.

Reply:
Agreed, I think we need to have another way to demonstrate provider use in a script. We can commit here for now, but it's a TODO item.